static/js/modules/tables.js

```diff
@@ -94,6 +110,51 @@ var committeeColumns = [
   {data: 'organization_type_full', className: 'min-desktop'},
 ];
 
+var filingsColumns = [
+  {
+    data: 'pdf_url',
+    className: 'all',
+    orderable: false,
+    render: function(data, type, row, meta) {
+      var anchor = document.createElement('a');
+      anchor.textContent = 'View filing';
+      anchor.setAttribute('href', data);
+      anchor.setAttribute('target', '_blank');
+      return anchor.outerHTML;
+    }
+  },
+  {data: 'committee_name', className: 'min-desktop'},
+  {data: 'amendment_indicator', className: 'min-desktop'},
+  {data: 'report_type_full', className: 'min-desktop'},
+  dateColumn({data: 'receipt_date', className: 'min-tablet'}),
+  currencyColumn({data: 'total_receipts', className: 'min-tablet'}),
+  currencyColumn({data: 'total_disbursements', className: 'min-tablet'}),
+  currencyColumn({data: 'total_independent_expenditures', className: 'min-tablet'}),
+];
+
+var filingsTableColumns = [
+  {
+    data: 'pdf_url',
+    className: 'all',
+    orderable: false,
+    render: function(data, type, row, meta) {
+      var anchor = document.createElement('a');
+      anchor.textContent = 'View filing';
+      anchor.setAttribute('href', data);
+      anchor.setAttribute('target', '_blank');
+      return anchor.outerHTML;
+    }
+  },
+  {data: 'committee_name', className: 'min-desktop', orderable: false},
```

Comment:
For the committee page, we don't need committee name, but for the filings query builder we do since you could query across committees. That's why there are two tables. You can see the filings query builder on the front end at `/filings`.